bitgouel-batch/src/main/kotlin/team/msg/job/student/GraduateStudentJobConfiguration.kt

```diff
@@ -0,0 +1,183 @@
+            .reader(graduateStudentItemReader())
+            .processor(graduateStudentItemProcessor())
+            .writer(graduateStudentItemWriter())
+            .listener(batchStepExecutionListener)
+            .build()
+    }
+
+    @Bean
+    @JobScope
+    fun clearStudentStep(): Step {
+        return stepBuilderFactory.get(JOB_NAME + "_clear_step")
+            .chunk<Student, Student>(CHUNK_SIZE)
+            .reader(graduateStudentItemReader())
+            .processor(clearStudentItemProcessor())
+            .writer(graduateStudentItemWriter())
+            .listener(batchStepExecutionListener)
+            .build()
+    }
+
+    @Bean
+    @StepScope
+    fun graduateStudentItemReader(): JdbcCursorItemReader<Student> {
+        logCohort()
+        return JdbcCursorItemReaderBuilder<Student>()
+            .sql("SELECT s " +
+                    "FROM STUDENT s WHERE s.studentRole = STUDENT " +
+                    "AND s.cohort = ${parameter.cohort}"
+            ).rowMapper(BeanPropertyRowMapper(Student::class.java))
+            .fetchSize(CHUNK_SIZE)
+            .dataSource(dataSource)
+            .name("graduateStudentReader")
+            .build()
+    }
+
+    @Bean
+    @StepScope
+    fun graduateStudentItemProcessor(): ItemProcessor<Student, Student> = ItemProcessor { newGraduateStudent(it) }
+
+    @Bean
+    @StepScope
+    fun clearStudentItemProcessor(): ItemProcessor<Student, Student> = ItemProcessor { clearStudent(it) }
+
+    @Bean
+    @StepScope
+    fun graduateStudentItemWriter(): JpaItemWriter<Student> {
+        val writer = JpaItemWriter<Student>()
+        writer.setEntityManagerFactory(entityManagerFactory)
+        writer.setUsePersist(false)
+        return writer
+    }
+
+    private fun newGraduateStudent(student: Student): Student {
+        return student.run {
+            Student(
+                id = id,
+                user = user,
+                club = club,
+                classRoom = classRoom,
+                grade = grade,
+                number = number,
+                cohort = cohort,
+                credit = credit,
+                studentRole = StudentRole.GRADUATE
+            )
+        }
+    }
+
+    private fun clearStudent(student: Student): Student {
+        return student.run {
+            Student(
+                id = id,
+                user = user,
+                club = club,
+                classRoom = classRoom,
+                grade = grade,
+                number = number,
+                cohort = cohort,
+                credit = credit,
+                studentRole = StudentRole.STUDENT
+            )
+        }
+    }
+
+    private fun logCohort() {
+        val startDateCohort = parameter.jobStartDate.year - 2023
+        val cohort = parameter.cohort
+
+        log.info("""
+            STUDENT READ를 시작합니다. 시작 날짜: ${parameter.jobStartDate}
+            졸업 예정 기수: ${startDateCohort}, parameter.cohort: $cohort
```

Comment:
```suggestion
            졸업 예정 기수: $startDateCohort, parameter.cohort: $cohort
```
굳이 필요 없는 {} 는 삭제하는 게 좋을 것 같아요